.ghal.rules.json

```diff
@@ -1,113 +1,275 @@
-              "(?i).*docs\/architecture\/microservices.*": ":book: guide - .NET Microservices",
-              "(?i).*docs\/architecture\/modern-web-apps-azure.*": ":book: guide - ASP.NET Core web apps",
-              "(?i).*docs\/architecture\/modernize-with-azure-containers.*": ":book: guide - Modernizing w/ Windows containers",
-              "(?i).*docs\/architecture\/serverless.*": ":book: guide - Serverless apps",
-              "(?i).*docs\/core.*": ":books: Area - .NET Core Guide",
-              "(?i).*docs\/core\/tools.*": ":card_file_box: Technology - CLI",
-              "(?i).*docs\/core\/docker.*": ":card_file_box: Technology - Docker",
-              "(?i).*docs\/csharp.*": ":books: Area - C# Guide",
-              "(?i).*docs\/desktop-wpf*": ":books: Area - Desktop Guide,:card_file_box: Technology - WPF",
-              "(?i).*docs\/framework.*": ":books: Area - .NET Framework Guide",
-              "(?i).*docs\/framework\/configure-apps\/file-schema\/network.*": ":card_file_box: Technology - NCL",
-              "(?i).*docs\/framework\/configure-apps\/file-schema\/wcf.*": ":card_file_box: Technology - WCF",
-              "(?i).*docs\/framework\/data\/adonet.*": ":card_file_box: Technology - Data Access",
-              "(?i).*docs\/framework\/data\/wcf.*": ":card_file_box: Technology - WCF",
-              "(?i).*docs\/framework\/docker.*": ":card_file_box: Technology - Docker",
-              "(?i).*docs\/framework\/install.*": ":card_file_box: Technology - Installers",
-              "(?i).*docs\/framework\/migration-guide.*": ":card_file_box: Technology - AppCompat",
-              "(?i).*docs\/framework\/network-programming.*": ":card_file_box: Technology - NCL",
-              "(?i).*docs\/framework\/windows-workflow-foundation.*": ":card_file_box: Technology - WF",
-              "(?i).*docs\/framework\/wpf.*": ":card_file_box: Technology - WPF",
-              "(?i).*docs\/framework\/wcf.*": ":card_file_box: Technology - WCF",
-              "(?i).*docs\/framework\/winforms.*": ":card_file_box: Technology - WinForms",
-              "(?i).*docs\/fsharp.*": ":books: Area - F# Guide",
-              "(?i).*docs\/machine-learning.*": ":books: Area - ML.NET Guide",
-              "(?i).*docs\/spark.*": ":books: Area - .NET for Apache Spark Guide",
-              "(?i).*docs\/standard.*": ":books: Area - .NET Guide",
-              "(?i).*docs\/standard\/security.*": ":card_file_box: Technology - Security",
-              "(?i).*docs\/standard\/standard\/design-guidelines\/.*": ":book: guide - Framework Design Guidelines",
-              "(?i).*docs\/visual-basic.*": ":books: Area - Visual Basic Guide"
+    "opened": {
+      "processor-default": {
+        "milestone-set": "![month]"
+      },
+      "processor-conditional": {
+        "type": "query",
+        "value": "PullRequest.base.ref == 'master'",
+        "processors": {
+          "processor-files": {
```

Comment:
@mairaw for reference, this section uses a new conditional processor that lets you run a query over the json schema of the PR, Issue, or Comment. So in this case, when a PR is opened it checks if the PR targets master and then proceeds to the nested processor which is the changed files detection which adds labels. 

This should solve your problem with opening PRs that target live.